app/src/lib/SourceData.svelte

```diff
@@ -0,0 +1,55 @@
+<script>
+	import Icon from '@iconify/svelte'
+	import { PUBLIC_SOURCES_API_HOST } from '$env/static/public'
+	import SemanticEncoding from './SemanticEncoding.svelte'
+	import TargetData from './TargetData.svelte'
+
+	/** @type {Reference} */
+	export let reference
+
+	/**
+	 * @param {Reference} reference
+	 *
+	 * @returns {Promise<SourceData>}
+	 */
+	async function get_source_data(reference) {
+		const response = await fetch(get_sources_url(reference))
+
+		return await response.json()
+	}
+
+	/**
+	 * @param {Reference} reference
+	 *
+	 * @returns {string} fully-qualified URL to the sources API
+	 */
+	function get_sources_url({ source, book, chapter, verse }) {
+		return `${PUBLIC_SOURCES_API_HOST}/${source}/${book}/${chapter}/${verse}`
+	}
+</script>
+
+{#await get_source_data(reference)}
+	<span class="loading loading-spinner text-warning" />
+	getting the source data...
+{:then source}
+	<h4 class="flex justify-between">
+		Phase 1 encoding
+
+		<a href={get_sources_url(reference)} target="_blank" class="link link-accent link-hover text-sm flex items-end">
+			all source details
+			<Icon icon="fe:link-external" class="h-6 w-6" />
+		</a>
+	</h4>
+	<p>
+		{source.phase_1_encoding}
+	</p>
+
+	<SemanticEncoding phase_2_encoding={source.phase_2_encoding} />
+
+	<h4>
+		Generated English text
+	</h4>
+	<p>
+		<TargetData {reference} />
```

Comment:
Just curious why this needs to wait for the source data to finish loading? They could be done in parallel. Up to you, not blocking by any means

Reply:
yeah, that's a good call, probably just that way by virtue of the way I refactored but you should be right.  I'll make that change to make sure there's not something I'm not thinking of.

Reply:
fixed those:  [061303e](https://github.com/presciencelabs/tabitha-ontology/pull/19/commits/061303e58b892b3329fa1f1b86e6ecf9c57e3fd7)